### ISSUE TITLE
Please help improve the performance on doing the comparing

### ISSUE THREAD

**Issue by neoyxm** [yousseb/meld#32]
Hi yousseb,
Thanks for your  great job. And I can enjoy the native meld on macOS, especially for the retina support(its really awesome).

On using the meld on Mac, i just met two issues related to performance:
1. A little bit slow to compare two files:  1500 line each file,  5 lines diff, cost about 1s to finish the comparing.
2. It's not so smooth to scroll up / down the document.

Here is my working environment:
**1. Device** : Mac book pro
- macOS Sierra 10.12.3
- MacBook Pro (Retina, 13-inch, Early 2015)
- 2.9 GHz Intel Core i5
- 8 GB 1867 MHz DDR3

**2. Python** 2.7.12
**3. Meld**  3.16.0

Thanks again for your contribution.
Neo Yang

**Comment by yousseb** [comment]
Sorry for the delay in response. I'm a bit swamped..
You may want to try the last 3.19.0 release. The GDK on this release is left to default - should allow hardware acceleration. Hopefully this may speed things up.

The bad news is that it has to use Python 3.6. Not sure how this would perform against the Python 2.7 with the 3.16 release.

**Comment by neoyxm** [comment]
HI  yousseb,

Thanks for your reply.
The 3.19.0 is much better than previous version.

I'm closing this issue now.

Thanks again.

**Comment by yousseb** [comment]
Glad to hear. Thanks!

**Comment by asimonov** [comment]
yes, thanks again for great work on Meld on Mac.
but 3.19.0 is sluggish for me on 10.13.4
does it have some pre-requisites like python?

**Comment by wuziq** [comment]
Meld 3.19 and 3.16 have been sluggish for me on Mac as well.

Things that have a delay (~300-500ms?  pretty significant):

- maximizing window (the screen area to be filled first turns black and is then filled when the window maximizes)
- scrolling (remains sluggish all throughout scrolling)
- switching between tabs
- opening / loading files

Things that don't have a delay:

- clicking the various buttons on the "New comparison" tab.

Using Macbook Pro
Siera 10.12.2
2.2 GHz Intel Core i7
16-GB 1600 MHz DDR3
Intel Iris Pro 1535 MB

I've tried disabling syntax highlighting, line numbers, and highlighting the current line.  I've tried using different fonts.  No difference.

Am I missing something?
